inscriptions/ISic1516.xml

```diff
@@ -177,17 +182,27 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI333765">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    πλίο <surplus>²⁶πλείω</surplus>²⁶ ἔλαττον κε, ἀνε-<lb n="2"/>παύσετο . . .
-                    <lb n="3"/>
+                    <lb n="1"/>Εὐσκία ἡ ἄ<hi rend="ligature">με</hi><hi rend="ligature">ντ</hi>ος <expan><abbr>ζή</abbr><ex>σ</ex><abbr>α</abbr><ex>σα</ex></expan>
+                    <lb n="2"/>χρηστῶς καὶ σε<hi rend="ligature">μνῶς</hi> ἔτη
+                    <lb n="3"/><expan><abbr>πλῖο</abbr><ex>ν</ex></expan> ἔλαττον <num value="25">κε</num> ἀνε
+                    <lb n="4" break="no"/>παύσετο <hi rend="ligature">τῇ</hi> ἑορ<hi rend="ligature">τῇ</hi> <hi rend="ligature">τῆ</hi>ς κυ
+                    <lb n="5" break="no"/>ρίας μου Λουκίας εἰς <hi rend="ligature">ἢν</hi>
+                    <lb n="6"/>οὔκ ἐστι<hi rend="ligature">ν ἐ</hi>νκώ<hi rend="ligature">με</hi>ιον
```

Comment:
Ah, this will be a problem later - a ligature across two words. Leave it like this for now, I'll think about what to do with it. Most likely split hi elements with `@part` start and end, or something like this.